content/tutorial-escrow.md

```diff
@@ -511,9 +511,19 @@ _Websocket_
 
 In the above example, `r3wN3v2vTUkr5qd6daqDc2xE4LSysdVjkT` is the sender of the escrow, and the increase in `Balance` from 99999**8**9990 drops to 99999**9**9990 drops represents the return of the escrowed 10,000 drops of XRP (0.01 XRP).
 
-## Look up escrows by sender
+**Tip:** If you don't know what `OfferSequence` to use in the [EscrowFinish transaction][] to execute an escrow, use the [`tx` method](reference-rippled.html) to look up the transaction that created the escrow, using the identifying hash of the transaction in the Escrow's `PreviousTxnID` field. Use the `Sequence` value of that transaction as the `OfferSequence` value when finishing the escrow.
+
+## Look up escrows
+
+All pending escrows are stored in the ledger as [Escrow objects](reference-ledger-format.html#escrow).
+
+You can look up escrow nodes by [sender](#look-up-escrows-by-sender) or [receiver](#look-up-escrows-by-receiver) using the [`account_objects`](reference-rippled.html#account-objects) method.
 
-All pending escrows are stored in the ledger as [Escrow objects](reference-ledger-format.html#escrow). You can look up escrow nodes owned by your address using the [`account_objects` method](reference-rippled.html#account-objects).
+###Look up escrows by sender
+
+You can use the [`account_objects`](reference-rippled.html#account-objects) method to look up escrows by sender.
+
+The following examples illustrate an escrow lookup by sender. Note that the `account` in the request is the sender `Account` of the escrow `account_objects` returned in the response.
```

Comment:
This may warrant a little more clarification. I think that the `account_objects` list (now) contains all escrows sent to or from an address, so you may get a bunch sent _by_ an address if you try to look up ones being sent _to_ that address.

The examples you have don't have a mix because you set them up that way, but in the wild, it's entirely possible for that to occur. Unfortunately, I don't think the API method has an easy way to filter, so you'll just have to tell people to check whether their address is the `Account` (sender) or `Destination` (recipient), or both, for each escrow in the response, when reading a response from `account_objects`.

Reply:
Yeppppp... got it. Thank you for taking the time to call this out. 💯  I oversimplified the samples. Indeed, in the wild it would/could not be as neat. 

I updated the samples to show a mix and have provided updated text in both the sender and receiver sections that I hope provides the necessary clarifications. 

I'm using the same samples in both sections. Likely the reader will not notice, but I'm hoping that it reinforces the use case of the "mixture" being returned. 

I'll be sure to delete the new code samples I added. I guess it wouldn't hurt to rename the code samples to not be sender specific.